crates/biome_js_analyze/src/lint/suspicious/no_redundant_use_strict.rs

```diff
@@ -22,10 +24,6 @@ declare_lint_rule! {
  ///
  /// Instead, `.cjs` files are considered "scripts" and the directive `"use strict"` is accepted and advised.
  ///
- /// Note that the leading trivia, e.g., comments or newlines preceding
- /// the redundant `"use strict"` will also be removed. So that comment
- /// directives won't be transferred to a wrong place.
```

Comment:
From this comment, it looks an intended behaviour. I am not sure about a reason for this, but may need some discussion before removing.

Reply:
> If we remove the directive, the comments don't have any more meaning.

I do think that's an assumption. It may be true in many or most cases, but a `"use strict"` may very well by preceeded by module documentation for instance, or a license comment, something that people want to have at the top before the `"use strict"` directive.

Removing such comments based on an assumption doesn't feel like a _safe_ fix to me.

Reply:
Just providing some historic facts :)

Reply:
We could what `organizeImports` does: keep comments at the start of the file.

Reply:
there are some tools for developers to add file description at the top of the source file:
- https://marketplace.visualstudio.com/items?itemName=cschlosser.doxdocgen#file-descriptions
- https://marketplace.visualstudio.com/items?itemName=psioniq.psi-header

removing both directive and comments may confuse users